src/fragments/sdk/push-notifications/android/getting-started.mdx

```diff
@@ -140,14 +142,21 @@ Use the following steps to connect your app to the push notification backend ser
             }
             return pinpointManager;
         }
+    }
+    ```
 
-        @Override
-        protected void onCreate(Bundle savedInstanceState) {
-            super.onCreate(savedInstanceState);
-            setContentView(R.layout.activity_main);
+1. Next, configure your application to use your custom `Application class`. Open the `AndroidManifest.xml` file located in your project directory at `app/src/main/AndroidManifest.xml`.
```

Comment:
does this need to be `2.`?  I can't remember if this works to create a list or not

Reply:
I left as 1. because there were multiple that listed as just 1. and when building, it looked correct.